CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 ### Fixes
 
 * **Clarify IAM Role Requirement for GCS Platform Connectors**. The GCS Source Connector requires Storage Object Viewer and GCS Destination Connector requires Storage Object Creator IAM roles.
+* **Change table extraction defaults** Change table extraction defaults in favor of using `skip_infer_table_types` parameter and reflect these changes in documentation.
```

Comment:
Could we update to `0.13.0` with this changes since we're changing default behavior on a feature that's heavily used?